docs/nvrm/pmu/ucode-cmds.rst

```diff
@@ -0,0 +1,295 @@
+.. _ucode-cmds:
+
+======================
+PMU microcode commands
+======================
+
+.. contents::
+
+
+Introduction
+============
+
+.. todo:: write me
+
+Sample Implementation
+------------------
+Example of setting up, running and handling potential error or timeout states.
+
+Pseudocode:
```

Comment:
Remove the :: from the next line, replace with 'Pseudocode::', drop one indentation level